.github/CODEOWNERS

```diff
@@ -0,0 +1,192 @@
+# This file lists the owners of code in different areas of the lnd codebase
+# Codeowners will own the review for the changes being merged into their
+# respective areas
+
+# aezeed
+/aezeed/ @yyforyongyu
+
+# alias manager
+/aliasmgr/ @yyforyongyu
+
+# amp
+/amp/ @yyforyongyu
+
+# auto pilot
+/autopilot/ @ellemouton @bhandras
```

Comment:
happy to be here but just a heads up that i have no experience here. Feels like perhaps @bitromortac would be a better fit here?